### PR TITLE
Quick Start Script

### LINKED ISSUE #494: Quick Start Script
**As a** potential user/operator of Zally
**I want** to have a quick start script/manual
**So that** I'm able to quickly start and try Zally components.

**Acceptance Criteria**
* Zally Server, database and Zally Web UI can be built and started with one script
* Documentation is adjusted

### PR DESCRIPTION
**Pull request by maxim-tschumak**
This PR includes:

* Docker-compose set up for Web UI, Server, and database
* Bash script for building the components and starting docker-compose
* Documentation adjustments

@netme Could you please test the script on OSX?

Fixes #494.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,23 +38,15 @@ The [Server Readme](server/README.md), [CLI Readme](cli/README.md) and [Web UI R
...
-
-# Build CLI tool
-go get github.com/zalando-incubator/zally/cli/zally
-cd $GOPATH/src/github.com/zalando-incubator/zally/cli/zally
-go build
-./zally lint /path/to/swagger/definition.yaml
+./build_and_run.sh
 ```
 
+Web UI is accessible on `http://localhost:8080`; Zally server on `http://localhost:8000`
```
Comment (rbarilani) [inline README.md]: @maxim-tschumak given docker-compose shouldn't be then webui on `3000` and  api on `8080` ?

**Comment by netme** [comment]
👍

**Comment by maxim-tschumak** [comment]
:+1:

**Comment by maxim-tschumak** [comment]
:+1:

**Comment by netme** [comment]
👍

**Pull request merged**